onnxruntime/core/platform/windows/env_time.cc

```diff
@@ -28,9 +28,12 @@ namespace {
 
 class WindowsEnvTime : public EnvTime {
  public:
+#if WINVER >= _WIN32_WINNT_WIN8
```

Comment:
This isnt a runtime check? Was it intended to be?

Reply:
No, it's intended to be a compile time check. If it's a build that must work downlevel (normal desktop build) it will call `GetModuleHandleW` to find `GetSystemTimePreciseAsFileTime` during runtime. If it's supposed to run in an appcontainer (which is always >= Win10, and always has `GetSystemTimePreciseAsFileTime`), it builds with GetSystemTimePreciseAsFileTime.